lib/logasm/tracer.rb

```diff
@@ -20,9 +20,9 @@ def initialize(logger)
     # Starts a new span.
     #
     # @param operation_name [String] The operation name for the Span
-    # @param child_of [SpanContext] SpanContext that acts as a parent to
+    # @param child_of [SpanContext, Span] SpanContext that acts as a parent to
     #        the newly-started Span. If a Span instance is provided, its
-    #        .span_context is automatically substituted.
+    #        context is automatically substituted.
```

Comment:
Does this mean you're now changing the interface from what OpenTracing defines, though?